src/models/room.js

```diff
@@ -205,12 +205,14 @@ Room.prototype.getLiveTimeline = function() {
  * <p>This is used when /sync returns a 'limited' timeline.
  *
  * @param {string=} backPaginationToken   token for back-paginating the new timeline
- * @param {boolean=} flush True to remove all events in all timelines. If false, only
- * the live timeline is reset.
+ * @param {string=} forwardPaginationToken token for forward-paginating the old live timeline,
+ * if absent or null, all timelines are reset.
```

Comment:
at first sight, this leads to a slightly weird-feeling API. Why does the absence of a forward-pagination token mean that we reset *all* timelines?

(I think it's plausible: it might just need some more explanation.)